tailwind.config.js

```diff
@@ -235,19 +235,25 @@ module.exports = {
 
   variants: {
     backgroundColor: [ 'responsive', 'rtl', 'hover', 'focus' ],
+    backgroundPosition: [ 'responsive', 'mirror' ],
+    borderColor: [ 'dark', 'hover', 'focus' ],
+    borderRadius: [ 'responsive', 'mirror' ],
```

Comment:
Is the `mirror` variant needed here?

Reply:
Yeah, I think we do want it for border-radius.  The first usage that jumps to my mind is in the chip component for the "X" button:

![Screen Shot 2020-05-28 at 11 01 10 AM](https://user-images.githubusercontent.com/17748358/83158466-d1b38180-a0d2-11ea-8660-bf8712cfe491.png)

Only one side is rounded, and it changes based on the "rtl" status